### PR TITLE
Add queries to Client which are required in React sdk

### PR DESCRIPTION
**Pull request by marsian83**
Adds following queries

- profile.getById
- profile.getUsers
- profile.getTotalGains
- profile.getMintFees
- escrow.getProtocolAndPlatformFees
- proposals.getByServiceId
- proposals.getByUser
- service.getSerivcePayments
- review.getByService

### CODE REVIEW

**Comment by marsian83** [comment]
please take a look @pranav-singhal 
During review of #1 Romain asked to have no queries in React sdk such that all queries are contained within the Client

File: packages/client/src/index.ts [hunk 1]
```diff
@@ -139,4 +143,36 @@ export class TalentLayerClient {
...
     );
   }
+
+  /**
+   * Provides access to user functionalities.
+   * @type {IUser}
+   */
+
+  // @ts-ignore
+  get user(): IUser {
```
Comment (0xRomain) [inline packages/client/src/index.ts]: user is called profile here to make it more "user friendly", so this already exist above

File: packages/client/src/proposals/graphql/index.ts [hunk 2]
```diff
@@ -35,3 +35,92 @@ export const getProposalById = (id: string) => `
...
+        expirationDate
+        platform {
+          id
+        }
+      }
+    }
+    `;
+
+export const getAllProposalsByUser = (id: string) => `
+          {
```
Comment (0xRomain) [inline packages/client/src/proposals/graphql/index.ts]: indent issue

File: packages/client/src/services/graphql/queries.ts [hunk 3]
```diff
@@ -1,5 +1,51 @@
 import { IProps } from '../types';
 
+export const getReviewsByService = (serviceId: string) => `
```
Comment (0xRomain) [inline packages/client/src/services/graphql/queries.ts]: review must be in dedicated client part

File: packages/client/src/services/graphql/queries.ts [hunk 4]
```diff
@@ -1,5 +1,51 @@
...
+    }
+    description{
+      id
+      content
+    }
+  }
+}
+`;
+
+export const getPaymentsByService = (serviceId: string, paymentType?: string) => {
```
Comment (0xRomain) [inline packages/client/src/services/graphql/queries.ts]: payments must be in escrow

File: packages/client/src/fees/graphql/index.ts [hunk 5]
```diff
@@ -0,0 +1,16 @@
+export const getProtocolAndPlatformsFees = (
+  originServicePlatformId: string,
+  originValidatedProposalPlatformId: string,
+) => `
+    {
+      protocols {
```
Comment (0xRomain) [inline packages/client/src/fees/graphql/index.ts]: i'm not sure about creating a dedicated folders for fees. They are all linked to another domain. 
Here I would say it's link to escrow

File: packages/client/src/fees/index.ts [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+    signatureApiUrl?: string,
+  ) {
+    this.graphQlClient = graphQlClient;
+    this.platformID = platformId;
+    this.ipfsClient = ipfsClient;
+    this.viemClient = viemClient;
+    this.signatureApiUrl = signatureApiUrl;
+  }
+
+  public async getMintFees(): Promise<any> {
```
Comment (0xRomain) [inline packages/client/src/fees/index.ts]: and here it's link to mint, so to profile domain

**Comment by marsian83** [comment]
Hi @0xRomaim, please review

File: packages/client/src/profile/graphql/index.ts [hunk 7]
```diff
@@ -30,3 +30,121 @@ export const getProfileByAddress = (address: Hash) =>
...
+                  rating
+                }
+              }
+              `;
+};
+
+export const getProfileById = (id: string) => `
+      {
+          user(id: "${id}") {
+          id
```
Comment (0xRomain) [inline packages/client/src/profile/graphql/index.ts]: check globally, you got some prettier issue

File: packages/client/src/profile/index.ts [hunk 8]
```diff
@@ -84,4 +92,51 @@ export class Profile {
       BigInt(fee),
     );
   }
+
+  public async getUsers(params: {
```
Comment (0xRomain) [inline packages/client/src/profile/index.ts]: naming just `getBy` sounds good for me, we will already be in the profile context 
so talentLayer.profile.getBy(...) is simpler and easier to understand

Comment (marsian83) [reply packages/client/src/profile/index.ts]: renamed

File: packages/client/src/profile/graphql/index.ts (hunk 7 above)
Comment (marsian83) [reply packages/client/src/profile/graphql/index.ts]: prettier does not work on strings, as we store these queries in backticked strings, prettier doesn't affect them
I have manually went through the files to format these (I have shifted to far rleft).

**Pull request merged**